### PR TITLE
[GUCCI] Expansion of techweb specific servers and configuration of servers for specific point generation, readds a forgotten thing for diseases and ORM tweak

### PR DESCRIPTION
**Pull request by ma44**
Sure does need some code review

### CODE REVIEW

File: code/controllers/subsystem/research.dm [hunk 1]
```diff
@@ -68,17 +79,26 @@ SUBSYSTEM_DEF(research)
...
-	bos_tech.add_point_list(bitcoins)
-	unknown_tech.add_point_list(bitcoins) //tbh these guys can get a fuckton of points, because it isn't even being used
+	if(multiserver_calculation)
+		science_tech.add_point_list(sciencecache)
+		bos_tech.add_point_list(boscache)
+		unknown_tech.add_point_list(unknowncache) //tbh these guys can get a fuckton of points, because it isn't even being used
+	else
+		science_tech.add_point_list(bitcoins)
+		bos_tech.add_point_list(bitcoins)
+		unknown_tech.add_point_list(bitcoins)
```
Comment (ma44) [inline code/controllers/subsystem/research.dm]: All of the above is horrible and has to be reworked unless you people want this monstrosity to be displayed for the world to see how bad web editor code can be made.

**Comment by AndrewL97** [comment]
@ma44 
stop combining multiple issues in the same branches please

File: code/datums/components/storage/concrete/bag_of_holding.dm [hunk 2]
```diff
@@ -1,4 +1,6 @@
 /datum/component/storage/concrete/bluespace/bag_of_holding/handle_item_insertion(obj/item/W, prevent_warning = FALSE, mob/living/user)
+	//STOP: GET SOME HELP
```
Comment (LetterN) [inline code/datums/components/storage/concrete/bag_of_holding.dm]: replace it with a bwoink instead

File: code/game/machinery/porta_turret/portable_turret.dm [hunk 3]
```diff
@@ -514,8 +499,12 @@
 
 /obj/machinery/porta_turret/proc/in_faction(mob/target)
 	for(var/faction1 in faction)
-		if(faction1 in target.faction)
-			return TRUE
+		if(islist(target.faction)
```
[the remark this answers is not in the document]
Comment (ma44) [reply code/game/machinery/porta_turret/portable_turret.dm]: >tfw compiles

**Comment by AndrewL97** [comment]
Tested - issue found + fixed, about to PR that fix

other than that, solid work.

**Pull request merged**